pkg/controller/volume/attachdetach/cache/desired_state_of_world.go

```diff
@@ -95,6 +96,9 @@ type DesiredStateOfWorld interface {
 	// GetPodToAdd generates and returns a map of pods based on the current desired
 	// state of world
 	GetPodToAdd() map[types.UniquePodName]PodToAdd
+
+	//IsPodVolumePreviouslyAdded checks whether pod references the specified volume is added to dsw
+	IsPodVolumePreviouslyAdded(podName volumetypes.UniquePodName, volumeSpec *volume.Spec, nodeName k8stypes.NodeName) (bool, error)
```

Comment:
For the sake of consistency, can you rename this to `PodExists(...)` and move it under `VolumeExists(...)`